game-category/index.md

```diff
@@ -0,0 +1,18 @@
+## Choosing a game category
+
+You have to choose a category for your game. The category you pick is entirely up to you. Categories are meant to be helpful, not restrictive. They are signposts to aid gamers in browsing the site and finding the games they want to play. You might want to browse the category listings yourself to see what kinds of games have been assigned to them in the past.
```

Comment:
We need to update the last sentence here as well as in "How do I add my game" you can't browse by category in wttf.